examples/cmake-examples/QmlBarcodeGenerator/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+cmake_minimum_required(VERSION 3.16.3)
+
+project(QmlBarcodeGenerator LANGUAGES CXX)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+#Check this file for any *_DIR variable definitions and other
+include("cmake/Locations.cmake")
+
+qt5_add_resources(RSCS qml.qrc)
```

Comment:
I know that we are not supporting the Qt 6 yet, but you can create the check for the version and use a different functions for adding resources depending if Qt 5 or Qt 6 is used. Just to make sure we won't miss that in the future.